model/src/test/kotlin/PackageCurationTest.kt

```diff
@@ -77,7 +77,7 @@ class PackageCurationTest : WordSpec({
 
             val curatedPkg = curation.apply(pkg.toCuratedPackage())
 
-            curatedPkg.pkg.apply {
+            with(curatedPkg.pkg) {
```

Comment:
Damn, you were a bit faster but I catched a bit more cases I believe: https://github.com/oss-review-toolkit/ort/pull/3048/commits

Reply:
I saw that case, too, but decided against converting it as it's slightly different (it's needed because `LicenseRule` is an inner class of `PackageRule`, not to group assertions). But I'm also fine with changing that case, too. Let's have @mnonnenmacher  decide 😉

Reply:
I agree with @sschuberth regarding `PackageRule`, I think in this case `with` makes it a bit harder to read and `apply` resembles the rules DSL a bit more. In general I think I would have settled for always using `apply` and never using `with`, because the style is closer to the other functions like `let` or `map` and I'm more used to it, and we use `apply` far more often.

Reply:
Ok, abandoned mine.